src/startup-tasks/task-start-consumers.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  name: 'Start event consumers',
```

Comment:
I think typically this is supposed to be the name instead of a description, but it doesn't really matter.